exchange/exchange.go

```diff
@@ -1245,12 +1245,12 @@ func (e *exchange) makeBid(bids []*entities.PbsOrtbBid, auc *auction, returnCrea
 	errs := make([]error, 0, 1)
 
 	for _, bid := range bids {
-		if !dsa.Validate(bidRequest, bid) {
-			RequiredError := openrtb_ext.ExtBidderMessage{
+		if err := dsa.Validate(bidRequest, bid); err != nil {
+			DSAMessage := openrtb_ext.ExtBidderMessage{
```

Comment:
Nitpick: Variable names should not start with a capital letter: `DSAMessage` -> `dsaMessage`